static/app/views/replays/detail/tagPanel/useTagFilters.spec.tsx

```diff
@@ -0,0 +1,62 @@
+import {browserHistory} from 'react-router';
+import type {Location} from 'history';
+
+import {reactHooks} from 'sentry-test/reactTestingLibrary';
+
+import {useLocation} from 'sentry/utils/useLocation';
+import useTagFilters, {
+  FilterFields,
+} from 'sentry/views/replays/detail/tagPanel/useTagFilters';
+
+jest.mock('react-router');
+jest.mock('sentry/utils/useLocation');
+
+const mockUseLocation = jest.mocked(useLocation);
+
+const tagFrame = TestStubs.ReplayRecord().tags;
+
+// {
+//   frames: {
+//     'browser.name': [ 'Other' ],
+//     'device.family': [ 'Other' ],
+//     environment: [ 'demo' ],
+//     'os.name': [ 'Other' ],
+//     platform: [ 'javascript' ],
+//     releases: [ '1.0.0', '2.0.0' ],
+//     'sdk.name': [ 'sentry.javascript.browser' ],
+//     'sdk.version': [ '7.1.1' ],
+//     'user.ip': [ '127.0.0.1' ]
+//   }
+// }
```

Comment:
It's more like this: 

```suggestion
// const tagFrame = {
//   'browser.name': [ 'Other' ],
//   'device.family': [ 'Other' ],
//   environment: [ 'demo' ],
//   'os.name': [ 'Other' ],
//   platform: [ 'javascript' ],
//   releases: [ '1.0.0', '2.0.0' ],
//   'sdk.name': [ 'sentry.javascript.browser' ],
//   'sdk.version': [ '7.1.1' ],
//   'user.ip': [ '127.0.0.1' ]
// }
```

but we might not need this comment, someone can go checkout `TestStubs.ReplayRecord().tags;` directly to see what the value is later on